packages/synapse-react-client/src/components/dataaccess/AccessSubmissionDashboard.integration.test.tsx

```diff
@@ -103,7 +102,7 @@ describe('AccessSubmissionDashboard tests', () => {
     )
   })
 
-  it.skip('Updates the passed props and URLSearchParams when updating arName', async () => {
```

Comment:
These tests were already skipped, but they seem less flaky when using the correct version of `history`